WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -104,9 +103,7 @@
     private static final int SELECT_LIBRARY_MENU_POSITION = 100;
     private static final int CLEAR_FEATURED_IMAGE_MENU_POSITION = 101;
 
-    private PostModel mPost;
-    private SiteModel mSite;
-
+    private PostSettingsListener mListener;
```

Comment:
We don't have to pass the site, but as long as we have the hook, I think it's good to get rid of that dependency. I don't see a good reason to not use the delegate here.

Just so we don't have a name that's too different than everything else, I'd recommend using `PostSettingsDelegate` here for the naming.